www/api/index.php

```diff
@@ -145,6 +145,18 @@ function if_is_admin($callback, $elseCallback=null) {
     ok();
 });
 
+$app->get('/v3_reachability', function() use ($app) {
+    $res = call_v3_api('HEAD', '/');
+
+    $app->response()->status($res[0]);
+
+    if ($res[0] === 200) {
+        ok($res[1]);
+    } else {
+        error('API unreachable!', 'This needs to be fixed immediately.');
```

Comment:
```suggestion
        error('api-unreachable', 'The V3 API is unreachable.');
```

I think the convention for these `error` calls is that the first parameter should be designed to be machine readable